### PR TITLE
Fixed #26201 -- Documented the consequences of rotating the CSRF token on login.

### PR DESCRIPTION
**Pull request by vehrlich**
https://code.djangoproject.com/ticket/26201

### CODE REVIEW

File: docs/ref/csrf.txt [hunk 1]
```diff
@@ -505,3 +504,12 @@ because it invalidates all previous forms. Most users would be very unhappy to
 find that opening a new tab on your site has invalidated the form they had
 just spent time filling out in another tab or that a form they accessed via
 the back button could not be filled out.
+
+Why I've get CSRF validation failure?
```
Comment (arnaudlimbourg) [inline docs/ref/csrf.txt]: Why did I get a CSRF validation failure?

File: docs/ref/csrf.txt [hunk 2]
```diff
@@ -505,3 +504,12 @@ because it invalidates all previous forms. Most users would be very unhappy to
...
+-------------------------------------
+
+New CSRF token is rotated with every login. All forms generated **before**
```
Comment (arnaudlimbourg) [inline docs/ref/csrf.txt]: CSRF tokens are rotated on login.

File: docs/ref/csrf.txt [hunk 3]
```diff
@@ -219,8 +219,7 @@ How it works
 
 The CSRF protection is based on the following things:
 
-1. A CSRF cookie that is set to a random value (a session independent nonce, as
-   it is called), which other sites will not have access to.
+1. A CSRF cookie that is set to a random value (a session independent nonce, as it is called, and it is rotated after login), which other sites will not have access to.
```
Comment (sergeykolosov) [inline docs/ref/csrf.txt]: I believe this should be wrapped

**Comment by timgraham** [comment]
merged in 369fa471f46cd517edf5fc82e4ef6138de3cff6e, thanks!

**Pull request closed without merging**